test/Dialect/FIRRTL/SFCTests/directories.fir

```diff
@@ -140,56 +140,38 @@ circuit TestHarness:
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.path_create instance %basepath @memNLA_0
-; MLIR_OUT:     om.list_create
-; MLIR_OUT:     om.object @MemorySchema
-; MLIR_OUT:     om.constant "bar_m_ext" : !om.string
-; MLIR_OUT:     om.constant #om.integer<2 : ui64> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<8 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<0 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.path_create instance %basepath @memNLA_1
-; MLIR_OUT:     om.path_create instance %basepath @memNLA_2
-; MLIR_OUT:     om.list_create
-; MLIR_OUT:     om.object @MemorySchema
-; MLIR_OUT:     om.constant "baz_m_ext" : !om.string
-; MLIR_OUT:     om.constant #om.integer<3 : ui64> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<8 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<0 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.constant #om.integer<1 : ui32> : !om.integer
-; MLIR_OUT:     om.class.fields %4, %20, %38 : !om.class.type<@MemorySchema>, !om.class.type<@MemorySchema>, !om.class.type<@MemorySchema>
+; MLIR_OUT-DAG: om.path_create instance %basepath @memNLA
+; MLIR_OUT-DAG: om.list_create
+; MLIR_OUT-DAG: [[OBJ1:%.+]] = om.object @MemorySchema
+; MLIR_OUT-DAG: om.constant "foo_m_ext" : !om.string
+; MLIR_OUT-DAG: om.constant #om.integer<1 : ui64> : !om.integer
+; MLIR_OUT-DAG: om.constant #om.integer<8 : ui32> : !om.integer
+; MLIR_OUT-DAG: om.constant #om.integer<1 : ui32> : !om.integer
+; MLIR_OUT-DAG: om.constant #om.integer<0 : ui32> : !om.integer
+; MLIR_OUT-DAG: om.path_create instance %basepath @memNLA_0
+; MLIR_OUT-DAG: om.list_create
+; MLIR_OUT-DAG: [[OBJ2:%.+]] = om.object @MemorySchema
+; MLIR_OUT-DAG: om.constant "bar_m_ext" : !om.string
+; MLIR_OUT-DAG: om.constant #om.integer<2 : ui64> : !om.integer
+; MLIR_OUT-DAG: om.path_create instance %basepath @memNLA_1
+; MLIR_OUT-DAG: om.path_create instance %basepath @memNLA_2
+; MLIR_OUT-DAG: om.list_create
+; MLIR_OUT-DAG: [[OBJ3:%.+]] = om.object @MemorySchema
+; MLIR_OUT-DAG: om.constant "baz_m_ext" : !om.string
+; MLIR_OUT-DAG: om.constant #om.integer<3 : ui64> : !om.integer
+; MLIR_OUT:     om.class.fields [[OBJ1]], [[OBJ2]], [[OBJ3]] : !om.class.type<@MemorySchema>, !om.class.type<@MemorySchema>, !om.class.type<@MemorySchema>
 
```

Comment:
This is fine. However, this test in appropriately combined directory testing with OM testing. (Apparently, my past self thought the same thing: https://github.com/llvm/circt/pull/5224#discussion_r1277128769)

If it's cleaner, in a follow-on, to either (1) factor this out of this test or (2) just delete this part of the test, then that would be good. @prithayan: Do you have any thoughts here?

Reply:
Yeah that would be great!

Reply:
Sorry for the messy lit tests and missing out your original review @seldridge , I think it makes sense to remove the testing from here. I added the om classes just because the original test was checking for the metadata verbatim ops, which will be replaced by the OM. I will take up the task to cleanup the tests.